src/indexEngines.ts

```diff
@@ -10,8 +10,8 @@ import { logger } from './utils/utils'
 // Initialize DB
 async function initDb(): Promise<void> {
   await setupDatabase(config.couchUri, ratesDbSetup)
-  ratesEngine().catch(e => logger('ratesEnginee failure', e))
-  uidEngine().catch(e => logger('uidEnginee failure', e))
+  // ratesEngine().catch(e => logger('ratesEnginee failure', e))
```

Comment:
turd here

Reply:
damn. the whole file change is a turd. how'd that sneak in there. will remove